src/components/tooltip/tooltip.js

```diff
@@ -442,9 +479,9 @@ function MdTooltipRegistry() {
 
   /**
    * Removes an event handler from the service.
-   * @param  {String} type Type of event handler.
-   * @param  {Function} handler The event handler itself.
-   * @param  {Boolean} useCapture Whether the event handler used event capturing.
+   * @param {string} type Type of event handler.
+   * @param {function} handler The event handler itself.
+   * @param {boolean} useCapture Whether the event handler used event capturing.
```

Comment:
lowercase function is also non-nullable by default. Capital Function is nullable by default.

Since you're using the lowercase one, you should be more specific about the params and return types.

`function(paramType): returnType`